python/pdo/test/storage.py

```diff
@@ -76,7 +77,7 @@ def verify_store_signature(store_response, duration, verifying_key) :
     result = client.store_blocks([block_data], duration=default_duration)
     assert result
 
-    assert verify_store_signature(result, default_duration, client.verifying_key)
+    assert (1 == verify_store_signature(result, default_duration, client.verifying_key))
 
```

Comment:
why are we using assert here? it pretty much blows up the entire application. can we just use the existing exceptions? there is already a very nice "SGX error"

Reply:
and what is the difference between the old version and the new version?